src/utils/setOperations.js

```diff
@@ -0,0 +1,11 @@
+class SetOperations {
```

Comment:
that can fit nicely inside the already existing `cellSetOperations.js` file in that folder, could you move the function there?

Reply:
nevermind, I just realised that this is for sets, not for cellSets